solidity/script/DeployHyperlane7683.s.sol

```diff
@@ -34,20 +34,20 @@ contract DeployHyperlane7683 is Script {
 
         vm.startBroadcast(deployerPrivateKey);
 
-        ProxyAdmin proxyAdmin = new OwnableProxyAdmin{salt: keccak256(abi.encode(ROUTER_SALT))}(proxyAdminOwner);
+        ProxyAdmin proxyAdmin = new OwnableProxyAdmin{ salt: keccak256(abi.encode(ROUTER_SALT)) }(proxyAdminOwner);
```

Comment:
Is it possible to separate the formatting changes into a seperate PR, that would make the diff easier to manage